backends/gstreamer/src/webrtc.rs

```diff
@@ -168,10 +165,7 @@ impl GStreamerWebRtcController {
                 None
             })
             .unwrap();
-        self.pipeline
-            .set_state(gst::State::Playing)
-            .into_result()
-            .unwrap();
+        self.pipeline.set_state(gst::State::Playing).unwrap();
```

Comment:
You probably want to handle errors here gracefully instead of panicking :)

Reply:
I'll leave this comment to @jdm and @Manishearth ;)

Reply:
Yeah, dealing with error handling better in webrtc code is a separate issue.